app/main.py

```diff
@@ -1,6 +1,29 @@
+import os
+from datetime import datetime as dt
+
+from dotenv import load_dotenv
+import requests
+
+load_dotenv()
+BASE_URL = "https://api.weatherapi.com/v1/current.json?"
+CITY = "Paris"
```

Comment:
You've done well by declaring the city as a constant. However, according to the checklist, the names of constants should be in uppercase. So, you should rename `CITY` to `CITY`.